CHANGELOG.md

```diff
@@ -12,6 +12,12 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## 7.6.0
+
+### Security
+
+* Removed reliance on `dotenv.find_dotenv` to prevent unintentional loading of environment files. Environment variable management is now more explicit to enhance security and reduce potential misconfigurations.
```

Comment:
```suggestion
* Disabled recursive search for .env file to prevent use of a potential threat vector by loading environment variables hosted in directories above the current working directory. Environment variable management is now more explicit to enhance security and reduce potential misconfigurations.
```